From 49af1667bd12563d041975fe58ad48a219fb6fe1 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Thu, 22 Sep 2022 15:48:03 +0100
Subject: [PATCH] trivial: Install the devhelp symlinks relative

We want /usr/share/docs/fwupd/libfwupd for the docs that can be referenced from
index.html, and a symlink of /usr/share/docs/libfwupd for devhelp.

This fixes the rpmbuild warning:

    # absolute symlink: /usr/share/doc/fwupd/libfwupd -> /usr/share/doc/libfwupd
    # absolute symlink: /usr/share/doc/fwupd/libfwupdplugin -> /usr/share/doc/libfwupdplugin
---
 docs/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/meson.build b/docs/meson.build
index 92b9e6fc7..75dd9a3d7 100644
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -82,10 +82,10 @@ if gidocgen_dep.found() and docs_python_deps.allowed() and gidocgen_app.found()
   #make devhelp work
   install_symlink('libfwupd',
     install_dir: join_paths(datadir, 'doc', 'fwupd'),
-    pointing_to: join_paths(datadir, 'doc', 'libfwupd'),
+    pointing_to: join_paths('..', 'libfwupd'),
   )
   install_symlink('libfwupdplugin',
     install_dir: join_paths(datadir, 'doc', 'fwupd'),
-    pointing_to: join_paths(datadir, 'doc', 'libfwupdplugin'),
+    pointing_to: join_paths('..', 'libfwupdplugin'),
   )
 endif
-- 
2.34.1

